gzip_logs_tools/log_compressor/log_compressor.c

```diff
@@ -885,10 +892,10 @@ main_thread(int argc, char *argv[])
 	pthread_t* cur_tinfo;
 	sigset_t set;
 	state_t* states;
-	unsigned thread_index;
-	unsigned arg_index;
+	unsigned  thread_index;
```

Comment:
redundant space here (the convention in this project is one)

Reply:
just to avoid the warnings later but whatever :)

Reply:
i'm ok with the type change, i think it's correct, comment was only about the spacing